core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/AbstractFromConfigFactory.java

```diff
@@ -77,8 +90,7 @@ protected GenuineVariableDescriptor<Solution_> deduceVariableDescriptor(
         return variableDescriptor;
     }
 
-    protected GenuineVariableDescriptor<Solution_> deduceVariableDescriptor(
-            EntityDescriptor<Solution_> entityDescriptor) {
+    protected GenuineVariableDescriptor<Solution_> getTheOnlyVariableDescriptor(EntityDescriptor<Solution_> entityDescriptor) {
```

Comment:
Dtto.

Reply:
This is being called once or twice when the CH phase is built. Unlike the case above, `entityDescriptor.getGenuineVariableDescriptorList()` is cheap.